src/sidebar/store/modules/real-time-updates.js

```diff
@@ -173,6 +173,20 @@ function hasPendingDeletion(state, id) {
   return state.pendingDeletions.hasOwnProperty(id);
 }
 
+/**
+ * @typedef RealTimeUpdatesStore
+ *
+ * // Actions
+ * @prop {typeof receiveRealTimeUpdates} receiveRealTimeUpdates
+ * @prop {typeof clearPendingUpdates} clearPendingUpdates
+ *
+ * // Selectors
+ * @prop {() => boolean} hasPendingDeletion
+ * @prop {() => Object<string, boolean>} pendingDeletions
+ * @prop {() => Object<string, Annotation>} pendingUpdates
```

Comment:
```suggestion
 * @prop {() => Object.<string, boolean>} pendingDeletions
 * @prop {() => Object.<string, Annotation>} pendingUpdates
```

Trivial change for stylistic consistency.

Reply:
agreed